### PR TITLE
Refactor evaluation due to new metrics structure

### PR DESCRIPTION
**Pull request by albert097**
* refactor of evaluators, due to new metrics structure
* separate POS tagging and sequence labeling
* change of task name `sequence tagging` to `sequence labeling`, because it is more popular https://en.wikipedia.org/wiki/Sequence_labeling

### CODE REVIEW

File: embeddings/pipeline/hugging_face_sequence_tagging.py [hunk 1]
```diff
@@ -35,5 +35,6 @@ def __init__(
         embedding = FlairTransformerWordEmbedding(embedding_name)
         task = SequenceTagging(output_path, hidden_size=hidden_size)
         model = FlairModel(embedding, task)
-        evaluator = SequenceTaggingEvaluator()
+        # todo: should be parametrised?
+        evaluator = POSTaggingEvaluator()
```
Comment (albert097) [inline embeddings/pipeline/hugging_face_sequence_tagging.py:39]: @ktagowski, usage of SequenceTaggingEvaluator/POSTaggingEvaluator should be parametrised? maybe a new pipeline class for POS?

File: embeddings/evaluator/metrics_evaluator.py [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
+    ) -> Sequence[Metric[Union[List[Any], np.ndarray, torch.Tensor], Dict[Any, Any]]]:
         pass
 
     def evaluate(self, data: Dict[str, np.ndarray]) -> Dict[str, Any]:
         return {
-            metric.name: metric.compute(
-                references=data["y_true"], predictions=data["y_pred"], **kwargs
-            )
-            for metric, kwargs in self.metrics
+            str(metric): metric.compute(data["y_true"], data["y_pred"]) for metric in self.metrics
```
Comment (ktagowski) [inline embeddings/evaluator/metrics_evaluator.py:21]: I would use keyword args like previously  `y_true=data["true"]` etc.

File: embeddings/pipeline/hugging_face_sequence_tagging.py (hunk 1 above)
Comment (ktagowski) [reply embeddings/pipeline/hugging_face_sequence_tagging.py:39]: For SequenceTaggingEvaluator probably there should be parametrization but I will test it with NER pipeline,  I like the idea o new pipeline class for POS

Comment (ktagowski) [reply embeddings/pipeline/hugging_face_sequence_tagging.py:39]: So in this PR I would skip parametrization of these metrics and make adapatation on real examples like NER / Political dataset in these task-related PR's.

**Review by laugustyniak — APPROVED** [review]

**Review by laugustyniak — DISMISSED** [review]

File: embeddings/pipeline/hugging_face_sequence_tagging.py (hunk 1 above)
Comment (albert097) [reply embeddings/pipeline/hugging_face_sequence_tagging.py]: i have already done this separation

File: embeddings/evaluator/metrics_evaluator.py (hunk 2 above)
Comment (albert097) [reply embeddings/evaluator/metrics_evaluator.py]: Kamil asked to pass args by key, rather than changing `y_true` to `true` (typo)

**Review by ktagowski — APPROVED** [review]

**Pull request merged**